server/mlabns/util/production_check.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def is_production_site(site_name):
+    """Determines if the given site name matches the production site schema
+
+    Args:
+        site_name: Name of site to check (e.g. "lga01")
+
+    Returns:
+        True if the site name matches the schema of a production site.
+    """
```

Comment:
This is probably OK.  We currently only have sites up to 05, but that could change and it might be reasonable to assume that the general form of the string is correct than to make assumptions about how many sites we have deployed in a given region.